setup-utils/install-mxnet-ubuntu-python.sh

```diff
@@ -5,7 +5,7 @@
 ######################################################################
 set -e
 
-MXNET_HOME="$HOME/mxnet/"
+MXNET_HOME=${PWD}"/../../mxnet/"
```

Comment:
one problem is that if i execute this .sh in $HOME with bash mxnet/setup-utils/install-mxnet-ubuntu-python.sh, then it will cause problem. 

one solution is use this sh's directory MYDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"